arangod/RestServer/DatabaseFeature.h

```diff
@@ -244,6 +248,7 @@ class DatabaseFeature : public ArangodFeature {
   bool _isInitiallyEmpty;
   bool _checkVersion;
   bool _upgrade;
+  replication::Version _defaultReplicationVersion;
```

Comment:
If possible I prefer inline initialization over constructor initializer lists, but feel free to ignore:
```suggestion
  replication::Version _defaultReplicationVersion = replication::Version::ONE;
```

Reply:
I will ignore, but with the rationale that I kept the code consistent with what was there before ;)